documentation/src/main/asciidoc/stories/assembly_cli_starting.adoc

```diff
@@ -3,7 +3,7 @@
 = Getting Started with {brandname} CLI
 The command line interface (CLI) lets you remotely connect to {brandname} Server to access data and perform administrative functions.
 Complete the following procedures to learn basic CLI usage such as creating users, connecting to {brandname}, and navigating resources.
-
+include::{topics}/proc_cli_use_container.adoc[leveloffset=+1]
```

Comment:
I don't think you can create users with the container (the CLI requires access to the server filesystem to create the user). Should the procedure be added to `proc_cli_connecting.adoc` instead?

Reply:
PS. We probably need an example of how to connect with an SSL-enabled server.

Reply:
I'm going to add more info about this. 
concerning SSL, that should definitely be covered in the SSL part

Reply:
So, I've checked all, and I moved this to the section of connecting. 
when we run the server in a container, we need to pass the admin and password credentials or some will be created for us. unless we override the server config. I don't think we need to add anything else here. however, I think we need to report the docs of the images to this documentation